system_tests/monitoring.py

```diff
@@ -138,8 +148,30 @@ def test_list_resource_descriptors(self):
             self.assertTrue(label.description)
 
     def test_query(self):
+        METRIC_TYPE = \
+            'pubsub.googleapis.com/topic/send_message_operation_count'
         client = monitoring.Client()
         query = client.query(METRIC_TYPE, hours=1)
         # There may be no data, but we can ask anyway.
         for _ in query:
             pass    # Not necessarily reached.
+
+    def test_create_and_delete_metric_descriptor(self):
+        NONCE = str(int(1000 * time.time()))
+        METRIC_TYPE = 'custom.googleapis.com/tmp/system_test_example_' + NONCE
+        METRIC_KIND = monitoring.MetricKind.GAUGE
+        VALUE_TYPE = monitoring.ValueType.DOUBLE
+        DESCRIPTION = 'System test example -- DELETE ME!'
+
+        client = monitoring.Client()
+        descriptor = client.metric_descriptor(
+            METRIC_TYPE,
+            metric_kind=METRIC_KIND,
+            value_type=VALUE_TYPE,
+            description=DESCRIPTION,
+        )
+
+        descriptor.create()
+        descriptor.delete()
+        with self.assertRaises(NotFound):
+            descriptor.delete()
```

Comment:
No need for any behavior, just want to avoid adding more flaky system tests. We already have quite a few (IIRC all due to eventual consistency) and I'm just trying to avoid introducing more.

If you like you could add a few sleeps and retries to tamp down the flakiness, or you could just wait until we come up with a solution that works for all of our flaky test cases.